source/optionTypes.ts

```diff
@@ -32,4 +32,7 @@ export interface eventOptions {
 }
 
 export type eventTypes =
-    'all' | 'selected' | 'births' | 'deaths' | 'events' | 'holidays'
\ No newline at end of file
+    'all' | 'selected' | 'births' | 'deaths' | 'events' | 'holidays'
+
+export type randomFormats =
+    'title' | 'summary' | 'html' | 'related' | 'mobile-sections' | 'mobile-sections-lead'
```

Comment:
Let's remove `html` for now because this will throw an error.

Reply:
Fixed in latest commit